apps/trustlab/src/payload/collections/Media.js

```diff
@@ -1,13 +1,21 @@
 import path from "path";
 import { fileURLToPath } from "url";
 
+import { createdBy } from "@commons-ui/payload";
+
+import { canManageContent } from "@/trustlab/payload/access/abilities";
+import { anyone } from "@/trustlab/payload/access/anyone";
+
 const filename = fileURLToPath(import.meta.url);
 const dirname = path.dirname(filename);
 
 const Media = {
   slug: "media",
   access: {
-    read: () => true,
+    read: anyone,
+    create: anyone,
```

Comment:
If `anyone` means a **logged in** user with any role then cool; otherwise we need to make sure only staff i.e. dashboard users can create `Media`, `Posts`, etc.